### PR TITLE
cli/doc improvements

### PR DESCRIPTION
**Pull request by caelean**
- new cli interface
- various readme/doc updates

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -4,14 +4,12 @@
 Modguard works by analyzing the abstract syntax tree (AST) of your codebase. The `Boundary` class and `@public` decorator have no runtime impact, and are detected by modguard statically. Boundary violations are detected at import time.
 ### Why does `modguard` live in my application code?
 Modguard is written as a Python library for a few main reasons:
-
+### What is a boundary?
+A **boundary** can be thought of as defining a logical module within your project. A project composed of decoupled logical modules with explicit public interfaces is easier to test and maintain.
```
Comment (emdoyle) [inline docs/faq.md:8]: This should move below the bullet points or above the enclosing question

File: modguard/show.py [hunk 2]
```diff
@@ -51,7 +51,7 @@ def _recurs_build_string(str_repr: str, level: int, current: Dict[str, Any]) ->
                 str_repr = _recurs_build_string(str_repr, level + 1, next_dict)
         return str_repr
 
-    return _recurs_build_string(str_repr, 0, dict_repr)[16:]
+    return _recurs_build_string(str_repr, 0, dict_repr) + "\n"
```
Comment (emdoyle) [inline modguard/show.py:54]: what is the change here?

**Review by emdoyle — APPROVED** [review]
Looks good, working well locally. CI action just needs to be updated for the new syntax

File: modguard/show.py (hunk 2 above)
Comment (caelean) [reply modguard/show.py:55]: I was trying to trim the leading newline and color settings but it wasn't working consistently, so instead I'm just putting a newline before and after the output

**Pull request merged**